frontend/src/components/SchemaList.vue

```diff
@@ -50,6 +51,12 @@
                 {{ schema.name }}
               </router-link>
             </div>
+            <div>
+              <i v-if="schema.deleted" class="eos-icons" data-bs-toggle="tooltip"
```

Comment:
We probably don't need to leave an empty `div` here, if the schema is not marked as deleted.
```suggestion
            <div v-if="schema.deleted">
              <i class="eos-icons" data-bs-toggle="tooltip"
```

Reply:
Right, currently not. With #69 this might change, but let's go with your suggestion until then.